leptos_i18n/src/routing.rs

```diff
@@ -16,7 +16,7 @@ use crate::{use_i18n_context, I18nContext, Locale};
 #[derive(Debug)]
 struct PathBuilder<'a>(Vec<&'a str>);
 
-impl<'a> Default for PathBuilder<'a> {
+impl Default for PathBuilder<'_> {
```

Comment:
It was hard errors on clippy and RA by using `#![deny(warnings)]` and I couldn't work without these changes. Even if you remove `#![deny(warnings)]`, it will emit too many warnings. Please let me know if you still persist on revert these changes.

Reply:
I have clippy and RA enabled and do not get those errors/warnings, do you have clippy pendantics enabled ?

Reply:
It seems those are warnings on latest nightly toolchain and `#![deny(warnings)]` make hard errors.

Reply:
I will address them separately.

Reply:
Ho right it's the nightly toolchain, I used to use nightly to avoid this situation, I just switched to nightly and got those errors, I'll takle them and push on the master don't worry.